src/test/java/api/CirculationRulesAPITests.java

```diff
@@ -67,4 +79,16 @@ private String getRulesText() {
     return text;
   }
 
+  private void setRules(String rules) {
+    createPoliciesIfDoNotExist(rules);
```

Comment:
I think it could make the intent of the tests clearer if the creation of policies wasn't hidden in a method whose intent is to update the rules.

Reply:
OK, it has been moved to setRules

Reply:
I'm not sure I follow. Maybe I didn't explain my comment well.

I'm suggesting that the tests would better express the scenario if the policies were created separately to when the rules are updated. At the moment, the policy creation is hidden, which I think obscures the test scenario.

Does that make sense? What do you think?

Reply:
Yes, I think it does. 
Updated